pom.xml

```diff
@@ -26,15 +25,15 @@
 
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.wso2.carbon.coordination</groupId>
-    <artifactId>org.wso2.carbon.cluster.coordinator.parent</artifactId>
+    <artifactId>carbon-coordination</artifactId>
```

Comment:
Artifact id should have org.wso2 part. Why is it removed?

Reply:
I followed the carbon datasources convention. Will change it back